Contents/mods/Perdition/media/lua/server/Items/Perdition_distributions.lua

```diff
@@ -14,7 +14,46 @@ local function preDistributionMerge()
                 "Perdition.CleaverUpgradeManual", 1,
                 "Perdition.ShovelUpgradeManual", 1,
             }
+        },
+        counter = {
```

Comment:
This is not a vanilla distribution.  Check the source file ProceduralDistributions.lua for distributions connected to vanilla containers.